scripts/other-yes-list/parse.ts

```diff
@@ -0,0 +1,80 @@
+import { genTextContextMatrix } from 'pdf-to-excel';
+import { readFile } from 'fs/promises';
+// import { readFileSync } from 'fs';
+
+// Original text matrix
+// const text = readFileSync('./ApprovedLiberalStudies.pdf');
+// genTextContextMatrix(text).then((table: any) => console.log(table.slice(22,100)));
+
+export type Course = {
+  /** Corresponds to the "prefix" column on the PDF */
+  subject: string;
+  /** Corresponds to the "code" column on the PDF */
+  catalogNbr: number;
+  /** The liberal studies categories satisfied by the course */
+  categories: readonly string[];
+};
+
+// parses PDF into matrix
+export async function getPDF() {
+  const pdfPath = await readFile('./ApprovedLiberalStudies.pdf');
+  return genTextContextMatrix(pdfPath);
+}
+
+// creates list of course entries
+export function genCourseEntries(data) {
+  // eslint-disable-next-line prefer-const
+  let courseEntries: Course[] = [];
```

Comment:
```suggestion
  const courseEntries: Course[] = [];
```
We should avoid disabling the linter if possible. Here, `courseEntries` is never reassigned, so it can be declared with `const`